omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -52,13 +52,21 @@
     # Whitelist the docs necessary to serve up error pages and friendly
```

Comment:
Let's change whitelist to "Allow" while we are here.

Reply:
Fixed